### ISSUE TITLE
Inputs in multiplayer should be ignored until gameplay start

### ISSUE THREAD

**Issue by bdach** [ppy/osu#11417]
This was especially visible in mania - if keys were pressed while the loading later was showing in `MultiplayerPlayer` (due to server not permitting gameplay start yet), the player hierarchy under the loading layer seemed to respond while it probably shouldn't (mania key lighting up, and I think key counter as well).

**Comment by peppy** [comment]
Not sure it's a huge issue to allow key input to work, but I agree that it feels a bit wrong if mouse input is blocked (in the case of rulesets which consume the cursor), so there should be parity of this behaviour at least.

**Comment by bdach** [comment]
It's a bit of an aesthetic/visual one as well, given that any animations initiated by inputs that happen while waiting for gameplay start don't actually play out until gameplay start is permitted (because the gameplay clock is paused).

**Comment by peppy** [comment]
Fair enough, they should definitely either play as expected or be disabled.
